ush/diag_statistics.py

```diff
@@ -13,36 +21,28 @@
 
 def get_diag_stats():
 
-    variables = ['sst', 't']
-
     tarfilename = 'gdas.t' + cyc + 'z.cnvstat'
-    tarfile = os.path.join(comout, tarfilename)
 
     os.chdir(data)
 
     for var in variables:
 
         diagfilename = 'diag_conv_' + var + '_ges.' + pdy + cyc + '.nc4'
-        diagfile = os.path.join(data, diagfilename)
         zipfilename = diagfilename + '.gz'
-        zipfile = os.path.join(data, zipfilename)
-        outfile = 'cnvstat.' + var + '.gdas.' + pdy + cyc
-
-        # TODO: these probably should be in the jjob. Will try to get them there
-        # once this is up and running
-        command = 'tar -xvf ' + tarfile + ' ' + zipfilename
-        print('running', command)
-        os.system(command)
-        command = 'gunzip ' + os.path.join(data, zipfile)
-        print('running', command)
-        os.system(command)
+        outfilename = 'cnvstat.' + var + '.gdas.' + pdy + cyc
```

Comment:
suggest ending this with `.csv` to make it clearer that it is a text file

Reply:
done